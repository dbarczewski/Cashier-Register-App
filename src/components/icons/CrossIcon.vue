<template>
  <svg width="24" height="24" viewbox="0 0 24 24">
    <path
      pid="0"
      d="M18.364 3.515L12 9.879 5.636 3.515c-.566-.566-1.556-.566-2.121 0-.566.565-.566 1.555 0 2.121L9.879 12l-6.364 6.364c-.566.566-.566 1.556 0 2.121.565.566 1.555.566 2.121 0L12 14.121l6.364 6.364c.566.566 1.556.566 2.121 0 .566-.565.566-1.555 0-2.121L14.121 12l6.364-6.364c.566-.566.566-1.556 0-2.121-.565-.566-1.555-.566-2.121 0z"
      fill="#BBB"
      fill-rule="nonzero"
    />
  </svg>
</template>

<script lang="ts">
import { defineComponent } from "vue";

export default defineComponent({
  name: "CrossIcon",
});
</script>
