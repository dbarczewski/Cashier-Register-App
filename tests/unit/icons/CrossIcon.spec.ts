import { shallowMount } from "@vue/test-utils";
import CrossIcon from "@/components/icons/CrossIcon.vue";

describe("icons/CrossIcon", () => {
  it("renders props.msg when passed", () => {
    const wrapper = shallowMount(CrossIcon);
    expect(wrapper.element).toMatchSnapshot();
  });
});
