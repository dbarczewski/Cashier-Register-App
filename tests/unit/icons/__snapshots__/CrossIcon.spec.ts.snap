// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icons/CrossIcon renders props.msg when passed 1`] = `
<svg
  height="24"
  viewbox="0 0 24 24"
  width="24"
>
  <path
    d="M18.364 3.515L12 9.879 5.636 3.515c-.566-.566-1.556-.566-2.121 0-.566.565-.566 1.555 0 2.121L9.879 12l-6.364 6.364c-.566.566-.566 1.556 0 2.121.565.566 1.555.566 2.121 0L12 14.121l6.364 6.364c.566.566 1.556.566 2.121 0 .566-.565.566-1.555 0-2.121L14.121 12l6.364-6.364c.566-.566.566-1.556 0-2.121-.565-.566-1.555-.566-2.121 0z"
    fill="#BBB"
    fill-rule="nonzero"
    pid="0"
  />
</svg>
`;
